internal/controller/isbservicerollout_controller_test.go

```diff
@@ -273,48 +274,51 @@ func Test_reconcile_isbservicerollout_PPND(t *testing.T) {
 	pipelineROReconciler = &PipelineRolloutReconciler{queue: util.NewWorkQueue("fake_queue")}
 
 	testCases := []struct {
-		name                    string
-		newISBSvcSpec           numaflowv1.InterStepBufferServiceSpec
-		existingISBSvcDef       *numaflowv1.InterStepBufferService
-		existingStatefulSetDef  *appsv1.StatefulSet
-		existingPipelineRollout *apiv1.PipelineRollout
-		existingPipeline        *numaflowv1.Pipeline
-		existingPauseRequest    *bool // was ISBServiceRollout previously requesting pause?
-		expectedPauseRequest    *bool // after reconcile(), should it be requesting pause?
-		expectedRolloutPhase    apiv1.Phase
```

Comment:
not sure why it's marking these entire blocks as changed, all I did was add the `initialInProgressStrategy`